CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.3.0 (2016-2-19)
```

Comment:
Closing this PR following the above steps and opening a new PR from pingread branch.